juju/juju-gui
# Update requirements.  Bump to 1.9.1.

QA: 

Run `make clean-all fast-babel qa-server` and ensure the gui comes up to the canvas w/o errors.

You'll need to grab the downloadcache from 

https://github.com/juju/juju-gui-downloadcache/pull/2
